Makefile

```diff
@@ -9,11 +9,14 @@ test_unit:
 
 lint:
 	flake8 .
```

Comment:
Is it still necessary to include `flake8`?

Reply:
Flake8 still does some useful things which are not covered by Black, e.g. detecting unused imports and variables, invalid format strings. FWIW at Brex, we use both and just disable any flake8 errors which conflict with Black (as I've done here)